src/main/java/ca/fxco/configurablepistons/blocks/pistons/basePiston/BasicPistonBaseBlock.java

```diff
@@ -329,6 +333,7 @@ public boolean canMoveBlock(BlockState state, Level level, BlockPos pos, Directi
             }
             if (!allowDestroy && !customBehavior.canDestroy(state))
                 return false;
+            return true;
```

Comment:
This bypasses the second `canMoveBlock` call, which will break the MBE piston.

Reply:
Only if `usesConfigurablePistonBehavior()` is true, although in that case `isMovable()` should be used instead.
However we might just want to remove  `customBehavior.isMovable` and only use `canMoveBlock` at this point.
Maybe move it into the interface so its easier to javadoc and expand on